documentation/asciidoc/computers/configuration/raspi-config.adoc

```diff
@@ -225,9 +225,9 @@ On the Raspberry Pi 4 and later, switch to the latest boot ROM software. Alterna
 
 ==== Wayland
 
-Switch between the X11 and Wayland backends. Raspberry Pi 4 and later use Wayland by default; other models of Raspberry Pi use X11 by default.
+Switch between the X11 and Wayland backends, and choose a window manager. Since Raspberry Pi OS _Bookworm_, all Raspberry Pi models run Wayland using labwc by default.
 
-NOTE: To use Wayland on Raspberry Pi models prior to Raspberry Pi 4, you must also add `wayland=on` to `/boot/firmware/cmdline.txt`.
+NOTE: To use Wayland on Raspberry Pi models prior to Raspberry Pi 4 running a version of Raspberry Pi OS earlier than _Bookworm_, add `wayland=on` to `/boot/firmware/cmdline.txt`.
```

Comment:
This isn't right. It's "older releases of Bookworm" where you'd have needed to add this `wayland=on`, not versions before Bookworm. Might be best to just delete this NOTE entirely?